src/main/java/ru/testtask/model/User.java

```diff
@@ -0,0 +1,54 @@
+package ru.testtask.model;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import javax.persistence.*;
+import java.util.Collection;
+import java.util.Set;
+
+@Document(collection = "users")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class User implements UserDetails {
+    @Id
+    private String id;
+    private String username;
```

Comment:
`username` should be unique filed

Reply:
Please add new test to `ProjectServiceTest` to check that impossible to create two projects with same names